app/models.py

```diff
@@ -92,8 +92,8 @@ def update_from_original(self, original):
 
 class ServiceUser(db.Model):
     __tablename__ = 'user_to_service'
-    user_id = db.Column(UUID(as_uuid=True), db.ForeignKey('users.id'), primary_key=True)
-    service_id = db.Column(UUID(as_uuid=True), db.ForeignKey('services.id'), primary_key=True)
+    user_id = db.Column(UUID(as_uuid=True), db.ForeignKey('users.id'), primary_key=True, nullable=True)
+    service_id = db.Column(UUID(as_uuid=True), db.ForeignKey('services.id'), primary_key=True, nullable=True)
```

Comment:
Is there protection in place to capture null (an empty string) where this code is called/used?

Reply:
How does changing this to `nullable=True` address the issue raised with the unexpected output?

Reply:
It doesn't make any sense to me that a mapping table (service to user) should allow null values even if that eliminates unexpected migration output.  It also doesn't make sense to me that primary keys should be nullable.

See this doc:
https://docs.sqlalchemy.org/en/13/core/metadata.html#sqlalchemy.schema.Column.params.nullable

I think the output is correct--primary keys should not be nullable.  I think the problem stems from the [original table specification](https://github.com/department-of-veterans-affairs/notification-api/blob/master/migrations/versions/0001_restart_migrations.py#L117) being incorrect.

Also note that using service_id and user_id as a composite primary key implies a 1-to-1 relationship.  Is that correct?  Can users not have multiple services?  Can services not be associated with multiple users?